samples/app-anonymous-users/nodejs/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-app-anonymous-users-nodejs",
+    "source": "officeDev",
+    "title": "Anonymous User Support",
+    "shortDescription": "This sample app shows anonymous users support in teams meeting apps.",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/app-anonymous-users/nodejs",
+    "longDescription": [
+      "This sample app shows anonymous users support in teams meeting apps."
+    ],
+    "creationDateTime": "2023-02-22",
+    "updateDateTime": "2023-02-24",
+    "products": [
+      "Teams"
+    ],
+    "metadata": [
+      {
+        "key": "TEAMS-SAMPLE-SOURCE",
+        "value": "OfficeDev"
+      },
+      {
+        "key": "TEAMS-SERVER-LANGUAGE",
+        "value": "javascript"
+      },
+      {
+        "key": "TEAMS-SERVER-PLATFORM",
+        "value": "express"
+      },
+      {
+        "key": "TEAMS-FEATURES",
+        "value": "bot"
+      }
+    ],
+    "thumbnails": [
+      {
+        "type": "image",
+        "order": 100,
+        "url": "https://raw.githubusercontent.com/OfficeDev/Microsoft-Teams-Samples/main/samples/app-anonymous-users/nodejs/Images/anonymoususersupport.gif",
+        "alt": "Solution UX showing anonymous user support in teams meeting apps"
```

Comment:
Solution UX showing anonymous user support in `Teams` meeting apps.

Reply:
Fixed